source/documentation/using_the_api/api_reference.md

```diff
@@ -8,6 +8,8 @@ View information about a register, including:
 * when the register was last updated
 * how many entries and records the register contains
 
+You can find the latest entry number by looking at the register information and comparing the most recent entry number with your own copy.
+
```

Comment:
Agree that this could be reworded slightly. Getting `total-entries` from this endpoint gives you the latest *entry number*. Comparing this number to the latest entry number you have locally tells you whether you are up to date with the register.